### PR TITLE
Add redirect

### LINKED ISSUE #6059: Move from Kanvas Visualizer to Kanvas Operator
### Desired Situation

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Description**

This PR fixes #6059

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 42d786faefdff0fd19dc4f8c9fe43be4260ba734 at: https://672991429053d5d92a136ab9--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 9017c694ff1fd327ad858ff7cc8925ad45e41a32 at: https://6729a128864776fc8828d879--layer5.netlify.app

File: src/pages/cloud-native-management/kanvas/operator.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import SEO from "../../../components/seo";
+
+import KanvasOperator from "../../../sections/Kanvas/Kanvas-visualize/index";
```
Comment (leecalcote) [inline src/pages/cloud-native-management/kanvas/operator.js:5]: Are we hyphenating or using snake case in our folder names?

File: gatsby-node.js [hunk 2]
```diff
@@ -308,6 +308,24 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
...
+  });
+  createRedirect({
+    fromPath: "/kanvas/visualize",
+    toPath: "/cloud-native-management/kanvas/operator",
+    redirectInBrowser: true,
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: "/kanvas/operator",
+    toPath: "/cloud-native-management/kanvas/operator",
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply gatsby-node.js:325]: So long as the opposing page uses "design", not "designer", that is.

File: src/sections/Kanvas/FeaturesSection/Visualize/VisualizerFeatures.js [hunk 3]
```diff
@@ -30,17 +30,17 @@ export default function VisualizerFeatures({ features }) {
     <VisualizerFeaturesWrapper>
       <Container className="visualizer-trigger-container">
         <div className="root test-container2">
-          <Link to="/cloud-native-management/kanvas/visualize">
+          <Link to="/cloud-native-management/kanvas/operator">
```
Comment (vishalvivekm) [inline src/sections/Kanvas/FeaturesSection/Visualize/VisualizerFeatures.js:33]:
```suggestion
          <Link to="/cloud-native-management/kanvas/operate">
```

File: src/sections/Kanvas/FeaturesSection/Visualize/VisualizerFeatures.js [hunk 4]
```diff
@@ -30,17 +30,17 @@ export default function VisualizerFeatures({ features }) {
...
+              <h1>Operate</h1>
               <div className="learn-more">
                 <h5 style={cursorOverArrow ? { color: "#ffffff" } : { color: "#ececec" }}>Learn more</h5>
                 <img src={cursorOverArrow ? LinkArrow : LinkArrowDark} alt="Learn more" style={{ maxWidth: "15%" }} className={cursorOverArrow ? "arrow-enter" : "arrow"} />
               </div>
             </div>
           </Link>
           <div className="g-grid-container contentContainer" id="add-border">
-            <Link to="/cloud-native-management/kanvas/visualize">
+            <Link to="/cloud-native-management/kanvas/operator">
```
Comment (vishalvivekm) [inline src/sections/Kanvas/FeaturesSection/Visualize/VisualizerFeatures.js:43]:
```suggestion
            <Link to="/cloud-native-management/kanvas/operate">
```

File: gatsby-node.js [hunk 5]
```diff
@@ -308,6 +308,24 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
     redirectInBrowser: true,
     isPermanent: true,
   });
+  createRedirect({
+    fromPath: "/cloud-native-management/kanvas/visualize",
+    toPath: "/cloud-native-management/kanvas/operator",
```
Comment (vishalvivekm) [inline gatsby-node.js:313]:
```suggestion
    toPath: "/cloud-native-management/kanvas/operate",
```

File: gatsby-node.js [hunk 6]
```diff
@@ -308,6 +308,24 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
...
+    redirectInBrowser: true,
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: "/kanvas/visualize",
+    toPath: "/cloud-native-management/kanvas/operator",
```
Comment (vishalvivekm) [inline gatsby-node.js:319]:
```suggestion
    toPath: "/cloud-native-management/kanvas/operate",
```

File: gatsby-node.js (hunk 2 above)
Comment (vishalvivekm) [inline gatsby-node.js:325]:
```suggestion
    toPath: "/cloud-native-management/kanvas/operate",
```

File: gatsby-node.js [hunk 7]
```diff
@@ -308,6 +308,24 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
...
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: "/kanvas/visualize",
+    toPath: "/cloud-native-management/kanvas/operate",
+    redirectInBrowser: true,
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: "/kanvas/operator",
```
Comment (vishalvivekm) [inline gatsby-node.js:324]:
```suggestion
    fromPath: "/kanvas/operate",
```

**Comment by l5io** [comment]
🚀 Preview for commit 238a68c99f1f4dbe9e5aa281f238b94a46bb247b at: https://672a340c491ba9b4e1dfbc62--layer5.netlify.app

File: src/pages/cloud-native-management/kanvas/operator.js (hunk 1 above)
Comment (vishalvivekm) [reply src/pages/cloud-native-management/kanvas/operator.js]: hyphenating

**Comment by l5io** [comment]
🚀 Preview for commit 910f1161dcaef2b1608b281d6db6c7d0d17c2808 at: https://672a90fcdd3a37563eb5386f--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 31d83053f9ba9688df60607b0fe382be5f453887 at: https://672bd455b965b8485fa8c1e5--layer5.netlify.app

**Pull request merged**